ui/pages/send/gas-display/gas-display.js

```diff
@@ -0,0 +1,337 @@
+  maxFeePerGas,
+  maxPriorityFeePerGas,
+  isMainnet,
+  showLedgerSteps,
+  isBuyableChain,
+  hexMinimumTransactionFee,
+  hexMaximumTransactionFee,
+  nativeCurrency,
+  chainId,
+  showAccountDetails,
+  gasError,
+}) {
+  const t = useContext(I18nContext);
+  const { estimateUsed, supportsEIP1559V2 } = useGasFeeContext();
+  const [showDepositPopover, setShowDepositPopover] = useState(false);
+  const currentProvider = useSelector(getProvider);
+  const networkName = NETWORK_TO_NAME_MAP[chainId];
+  const isInsufficientTokenError =
+    draftTransaction?.amount.error === INSUFFICIENT_TOKENS_ERROR;
+  const renderTotalMaxAmount = () => {
+    if (
+      primaryTotalTextOverride === undefined &&
+      secondaryTotalTextOverride === undefined &&
+      draftTransaction.asset.type === 'NATIVE'
+    ) {
+      // Native Send
+      return (
+        <UserPreferencedCurrencyDisplay
+          type={PRIMARY}
+          key="total-max-amount"
+          value={addHexes(
+            draftTransaction.amount.value,
+            hexMaximumTransactionFee,
+          )}
+          hideLabel={!useNativeCurrencyAsPrimaryCurrency}
+        />
+      );
+    }
+    // Token send
+    return useNativeCurrencyAsPrimaryCurrency
+      ? primaryTotalTextOverrideMaxAmount
+      : secondaryTotalTextOverride;
+  };
+  const renderTotalDetailTotal = () => {
+    if (
+      primaryTotalTextOverride === undefined &&
+      secondaryTotalTextOverride === undefined &&
+      draftTransaction.asset.type === 'NATIVE'
+    ) {
+      return (
+        <div className="confirm-page-container-content__total-value">
+          <LoadingHeartBeat
+            estimateUsed={draftTransaction?.userFeeLevel ?? estimateUsed}
+          />
+          <UserPreferencedCurrencyDisplay
+            type={PRIMARY}
+            key="total-detail-value"
+            value={hexTransactionTotal}
+            hideLabel={!useNativeCurrencyAsPrimaryCurrency}
+          />
+        </div>
+      );
+    }
+    return useNativeCurrencyAsPrimaryCurrency
+      ? primaryTotalTextOverrideMaxAmount
+      : secondaryTotalTextOverride;
+  };
+  const renderTotalDetailText = () => {
+    if (
+      primaryTotalTextOverride === undefined &&
+      secondaryTotalTextOverride === undefined
+    ) {
+      return (
+        <div className="confirm-page-container-content__total-value">
+          <LoadingHeartBeat
+            estimateUsed={draftTransaction?.userFeeLevel ?? estimateUsed}
+          />
+          <UserPreferencedCurrencyDisplay
+            type={SECONDARY}
+            key="total-detail-text"
+            value={hexTransactionTotal}
+            hideLabel={Boolean(useNativeCurrencyAsPrimaryCurrency)}
+          />
+        </div>
+      );
+    }
+    return useNativeCurrencyAsPrimaryCurrency
+      ? secondaryTotalTextOverride
+      : primaryTotalTextOverride;
+  };
+  const renderGasDetailsItem = () => {
+    return (
+      <div className="transaction-item">
```

Comment:
Why does this TransactionDetailItem get wrapped in 'transaction-item' div?

Reply:
@brad-decker It is not needed this 'transaction-item' div here. I deleted it [here](https://github.com/MetaMask/metamask-extension/pull/14634/commits/30e2c90ae66cf2a12aeb11e5ea35daa7cfd60340).